Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup Label="Signing">
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\build\keyfile.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
```

Comment:
><SignAssembly>true</SignAssembly> [](start = 4, length = 33)

The removal of DelaySign was intentional. The key used in this project is not secret. I believe we were told to do it this way for a .NET Core SDK project. The digital cert on official builds is the proof those come from Microsoft.

Reply:
From [.NET Core github documentation](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md#2-i-really-really-need-to-strong-name-what-kinds-of-issues-will-i-run-into):

> Firstly, we would recommend that these open source projects check-in their private key (remember, strong-names are used for identity, and not for security). 


---
In reply to: [354995850](https://github.com/Azure/azure-relay-dotnet/pull/124#discussion_r354995850) [](ancestors = 354995850)